docker/jupyter_pylibROM/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# start from pylibrom docker container
+FROM --platform=linux/amd64 ghcr.io/llnl/pylibrom/pylibrom_env:latest

```

Comment:
It might be better to leave out the `--platform` option here since we also want to support building for ARM as well.